internal/host/instructionmanager.go

```diff
@@ -80,10 +82,10 @@ func NewInstructionManager(log logrus.FieldLogger, db *gorm.DB, hwValidator hard
 		},
 		addHostsClusterToSteps: stateToStepsMap{
 			models.HostStatusKnown:                {[]CommandGetter{connectivityCmd}, defaultNextInstructionInSec},
```

Comment:
let's add the apiVIP check to Known state also. in case host spends a lot of time in known state we want api vip connectivity to continue be checked

Reply:
sure, added.